book.rb

```diff
@@ -0,0 +1,10 @@
+# Book class
+class Book
+  attr_accessor :title, :author, :rentals
+
```

Comment:
- [ ] A _rental_ can have many _books_, and _books_ can be subjected to many _rentals_. Let's therefore establish a _many-to-many_ association between _rentals_ and _books_ (let's bear in mind that the association between _person_ and _rentals_ is also _many-to-many_) by defining a method for adding rentals within the book object like below:

```ruby
class Book
  attr_accessor :title, :author, :rentals
  .
  .

  def add_rental(person, date)
    Rental.new(date, self, person)
  end
end
```